src/sql/workbench/contrib/resourceViewer/browser/resourceViewerEditor.ts

```diff
@@ -43,6 +54,10 @@ export class ResourceViewerEditor extends EditorPane {
 		const header = this.createHeader();
 		const tableContainer = this.createResourceViewerTable();
 
+		this._register(this._resourceViewerTable.onContextMenu(e => {
+			this.showContextMenu(e.anchor, e.item);
```

Comment:
Could you please check whether this event will be fired for header row as well? if so, need to make sure e.item is not undefined before calling the showContextMenu method.

Reply:
Nope - not called for header clicks.